### ISSUE TITLE
Program crashed with signal 11 with Nvidia Driver Version: 455.45.01 (prime-run)

### ISSUE THREAD

**Issue by ubley09** [godotengine/godot#44849]
**Godot version:**
Godot_v3.2.3-stable_x11.64

**OS/device including version:**
OS: Manjaro Linux x86_64 
Kernel: 5.10.2-2-MANJARO <- I tried with every kernel, doesn't matter.
GPU: NVIDIA GeForce GTX 1050 Ti Mobile 
GPU: Intel HD Graphics 630 
NVIDIA-SMI 455.45.01 <- Only with this driver, but I need this version on Kernel 5.10
Driver Version: 455.45.01
CUDA Version: 11.1

**Issue description:**
I launch Godot with prime-run command to run the editor on dedicated gpu. When I click Edit on ANY project, almost immediately crash to desktop. If I don't use prime-run, work fine, but on integrated gpu, doesn't run well 3D graphics.
I tried to set the project to GLES 2, doesn't matter. I tried install Godot from various places, doesn't matter.
If I click to run project, that work. I think there is a problem with the nvidia driver v455, but this only happen with Godot.

Error log:
```
- prime-run ./Godot_v3.2.3-stable_x11.64 -e --verbose
                     
Godot Engine v3.2.3.stable.official - https://godotengine.org
XInput: Refreshing devices.
XInput: No touch devices found.
Detecting GPUs, set DRI_PRIME in the environment to override GPU detection logic.
Only one GPU found, using default.
Failed loading custom cursor: X_cursor
Using GLES3 video driver
OpenGL ES 3.0 Renderer: GeForce GTX 1050 Ti/PCIe/SSE2
PulseAudio: detected 2 channels
PulseAudio: audio buffer frames: 512 calculated latency: 11ms
 
CORE API HASH: 17947557007960061372
EDITOR API HASH: 513236543915789544
Loading resource: /home/ubley/.config/godot/editor_settings-3.tres
EditorSettings: Load OK!
EditorSettings: Save OK!
Loaded builtin certs
Editing project: /home/ubley/Documents/Projects/Game/Godot/Projects/My3DGame (::home::ubley::Documents::Projects::Game::Godot::Projects::My3DGame)
EditorSettings: Save OK!
Godot Engine v3.2.3.stable.official - https://godotengine.org
XInput: Refreshing devices.
XInput: No touch devices found.
Detecting GPUs, set DRI_PRIME in the environment to override GPU detection logic.
Only one GPU found, using default.


Failed loading custom cursor: X_cursor
Using GLES3 video driver
OpenGL ES 3.0 Renderer: GeForce GTX 1050 Ti/PCIe/SSE2
handle_crash: Program crashed with signal 11
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /usr/lib/libc.so.6(+0x3d6a0) [0x7fb3ed3246a0] (??:0)
[2] /usr/lib/libnvidia-glcore.so.455.45.01(+0x1149d45) [0x7fb3eaef4d45] (??:0)
[3] /usr/lib/libnvidia-glcore.so.455.45.01(+0x114b00c) [0x7fb3eaef600c] (??:0)
[4] /usr/lib/libnvidia-glcore.so.455.45.01(+0xd64fc2) [0x7fb3eab0ffc2] (??:0)
[5] /usr/lib/libnvidia-glcore.so.455.45.01(+0xd6523d) [0x7fb3eab1023d] (??:0)
[6] /usr/lib/libnvidia-glcore.so.455.45.01(+0xd67c6c) [0x7fb3eab12c6c] (??:0)
[7] /run/media/ubley/HDD/Programs/Godot_v3.2.3-stable_x11.64() [0x23b80de] (<artificial>:?)
[8] /run/media/ubley/HDD/Programs/Godot_v3.2.3-stable_x11.64() [0x23ba75b] (<artificial>:?)
[9] /run/media/ubley/HDD/Programs/Godot_v3.2.3-stable_x11.64() [0x10163b4] (<artificial>:?)
[10] /run/media/ubley/HDD/Programs/Godot_v3.2.3-stable_x11.64() [0x2a834fb] (<artificial>:?)
[11] /run/media/ubley/HDD/Programs/Godot_v3.2.3-stable_x11.64() [0x86029d] (??:?)
[12] /usr/lib/libc.so.6(__libc_start_main+0xf2) [0x7fb3ed30f152] (??:0)
[13] /run/media/ubley/HDD/Programs/Godot_v3.2.3-stable_x11.64() [0x86b68e] (??:?)
-- END OF BACKTRACE --
```

**Steps to reproduce:**
Install Nvidia 455 driver
prime-run godot
Open any project to edit.

**Minimal reproduction project:**
Any project, doesn't matter.

**Comment by ubley09** [comment]
I was able to temporarily solve the problem. I downloaded the Godot Windows version and run with help of Lutris on dedicated gpu. But I really hope there will be some better solution for this.

**Comment by ubley09** [comment]
Today released the Nvidia 460.32.03 driver and the Godot run fine with prime-run!
